tracker/src/main/kotlin/org/dhis2/tracker/relationships/model/RelationshipViewModel.kt

```diff
@@ -0,0 +1,92 @@
+package org.dhis2.tracker.relationships.model
+
+import org.hisp.dhis.android.core.common.Geometry
+import org.hisp.dhis.android.core.common.ObjectStyle
+import org.hisp.dhis.android.core.relationship.Relationship
+import org.hisp.dhis.android.core.relationship.RelationshipType
+import java.util.Date
+
+data class RelationshipViewModel(
```

Comment:
Do you think could be confusing have a class named RelationshipViewModel and the viemodel RelationshipsViewModel? Maybe the data class could be RelationshipModel?

Reply:
Yes you are right